apps/users/models.py

```diff
@@ -0,0 +1,28 @@
+import uuid
+
+from django.db import models
+from django.contrib.auth.models import AbstractBaseUser
+from django.contrib.auth.models import PermissionsMixin
+from django.utils.translation import gettext_lazy as _
+from django.utils import timezone
+
+from .managers import UserManager
+
+
+class User(AbstractBaseUser, PermissionsMixin):
+    uuid = models.UUIDField(default=uuid.uuid4, unique=True)
+
+    email = models.EmailField(_("email address"), unique=True)
+    username = models.EmailField(null=True)  # field is required for python social auth
+
+    is_staff = models.BooleanField(default=False)
```

Comment:
What's the purpose of is_staff?

Reply:
Django requires the User model to have a base set of fields

https://docs.djangoproject.com/en/3.0/ref/contrib/auth/#django.contrib.auth.models.User